pkg/services/authz/legacy/server/cache.go

```diff
@@ -0,0 +1,38 @@
+package server
+
+import (
+	"time"
+
+	"github.com/grafana/grafana/pkg/infra/localcache"
+)
+
+const (
+	cacheTTL             = 1 * time.Minute
+	cacheCleanupInterval = 5 * time.Minute
+)
+
+func newCache[T any]() *cache[T] {
+	return &cache[T]{inner: localcache.New(cacheTTL, cacheCleanupInterval)}
+}
+
+type cache[T any] struct {
```

Comment:
Do you think it makes sense to use remote cache here? Are we going to run more than one instance of authz service?

Reply:
I mean this cache implements the `RemoteCache` interface, so potentially we could switch to remote cache, but I'm not sure it does make sense.